[2024-10-16T09:40:09.035+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [queued]>
[2024-10-16T09:40:09.091+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [queued]>
[2024-10-16T09:40:09.092+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:09.092+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:40:09.093+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:09.122+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-09-03 00:00:00+00:00
[2024-10-16T09:40:09.141+0000] {standard_task_runner.py:55} INFO - Started process 4237 to run task
[2024-10-16T09:40:09.152+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-09-03T00:00:00+00:00', '--job-id', '369', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmpjkafexfx']
[2024-10-16T09:40:09.158+0000] {standard_task_runner.py:83} INFO - Job 369: Subtask insert_rate
[2024-10-16T09:40:09.400+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:40:09.575+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:40:09.740+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-09-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-03T00:00:00+00:00
[2024-10-16T09:40:09.743+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-09-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:40:09.773+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:10.423+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:10.442+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-09-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:40:10.448+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedColumn: column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-09-03')
                              ^

[2024-10-16T09:40:10.472+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240903T000000, start_date=20241016T094009, end_date=20241016T094010
[2024-10-16T09:40:10.525+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 369 for task insert_rate (column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-09-03')
                              ^
; 4237)
[2024-10-16T09:40:10.579+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-16T09:40:10.649+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T09:49:51.749+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [queued]>
[2024-10-16T09:49:51.763+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [queued]>
[2024-10-16T09:49:51.764+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:51.764+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:49:51.765+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:49:51.792+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-09-03 00:00:00+00:00
[2024-10-16T09:49:51.799+0000] {standard_task_runner.py:55} INFO - Started process 7030 to run task
[2024-10-16T09:49:51.805+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-09-03T00:00:00+00:00', '--job-id', '1224', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmpnhlarwur']
[2024-10-16T09:49:51.807+0000] {standard_task_runner.py:83} INFO - Job 1224: Subtask insert_rate
[2024-10-16T09:49:51.933+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-09-03T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:49:52.001+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:49:52.083+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-09-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-03T00:00:00+00:00
[2024-10-16T09:49:52.087+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 87.57420135, '2024-09-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:49:52.100+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:52.478+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:49:52.489+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 87.57420135, '2024-09-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:49:52.492+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:49:52.509+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20240903T000000, start_date=20241016T094951, end_date=20241016T094952
[2024-10-16T09:49:52.559+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:49:52.589+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
